main.js

```diff
@@ -1 +1,109 @@
+      badge4: 'HTML',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+      buttons1: 'link',
+      button2: 'link2',
+    },
+    {
+      title: 'Keeping track of hundreds of components - 5',
+      image: 'img/test.png',
+      badge1: 'Ruby on rails',
+      badge2: 'CSS',
+      badge3: 'JavaScript',
+      badge4: 'HTML',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+      buttons1: 'link',
+      button2: 'link2',
+    },
+    {
+      title: 'Keeping track of hundreds of components - 6',
+      image: 'img/test.png',
+      badge1: 'Ruby on rails',
+      badge2: 'CSS',
+      badge3: 'JavaScript',
+      badge4: 'HTML',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+      button1: 'link',
+      button2: 'link2',
+    },
+  ];
+  function toggleBarWorks(workNumber) {
+    const y = document.getElementById('mobile-popup-section');
+    const i = workNumber;
+    document.getElementById('img-pop').src = projects[i].image;
+    document.getElementById('h3-popup').innerHTML = projects[i].title;
+    document.getElementById('badge1').innerHTML = projects[i].badge1;
+    document.getElementById('badge2').innerHTML = projects[i].badge2;
+    document.getElementById('badge3').innerHTML = projects[i].badge3;
+    document.getElementById('badge4').innerHTML = projects[i].badge4;
+    document.getElementById('p-pop').innerHTML = projects[i].description;
+    document.getElementById('btn-1').href = projects[i].button1;
+    document.getElementById('btn-2').href = projects[i].button2;
+
+    if (y.style.display === 'flex') {
+      y.style.display = 'none';
+    } else {
+      y.style.display = 'flex';
+    }
+  }
```

Comment:
## kindly work on it on the mobile version as well 🔥🙏🏻
- [ ]  Well done styling the popup window so far. However I noticed that you need to decrease the height a little bit so I can see the whole card because I can't see the buttons for see project and live version and the description as well. In my opinion you can make scrolling in the side or decrease the height for the pop up card a little bit so please work on it. 🔥😊🙏🏻

- [ ] In the mobile version, Kindly align the heading to not be very close to the side so it is like that I will drwa on it to see it below so you can add some padding to it

- Your design
![image](https://user-images.githubusercontent.com/58553711/188769870-fd1c3927-8a1a-4813-aaec-e92a7cb211a8.png)


- Figma design
![image](https://user-images.githubusercontent.com/58553711/188768197-f2219d90-6f33-4994-9eb4-ee0600d366fe.png)

- Mobile version

![image](https://user-images.githubusercontent.com/58553711/188770794-ab63debd-6cec-4241-a3b3-31dea36bbd6e.png)